### PR TITLE
Further adjust draft structure

### PR DESCRIPTION
**Pull request by JKRhb**
[Latest rendered version](https://namib.me/draft-coap-mud/structure/draft-romann-mud-constrained.html)

### CODE REVIEW

File: draft-romann-mud-constrained.md [hunk 1]
```diff
@@ -72,7 +72,11 @@ Building upon the terminology defined in {{!RFC8520}}, this specification introd
 
 TODO. (Remove if there are no additional terms.)
 
-# Exposing a MUD URL using NDP
+# Exposing a MUD URL in Constrained Environments
```
Comment (pulsastrix) [inline draft-romann-mud-constrained.md]: Isn't this basically the subject of the entire draft? I am unsure whether this heading might be redundant.

Comment (JKRhb) [reply draft-romann-mud-constrained.md]: Hmm, yeah, you're right. How about something like "Additional Methods for Exposing MUD URLs"?

Comment (pulsastrix) [reply draft-romann-mud-constrained.md]: I'd just remove the heading, other RFCs seem to be similarly structured, and nesting too many headings makes finding sections harder.

Comment (JKRhb) [reply draft-romann-mud-constrained.md]: Okay, sounds reasonable, I reverted this change :)

**Pull request merged**